packages/devtools/lib/src/timeline/frames_bar_plotly.dart

```diff
@@ -17,8 +18,10 @@ class FramesBarPlotly {
   static const int CPU_HIGH_WATER_MARK_TRACE = 4;
   static const int GPU_HIGH_WATER_MARK_TRACE = 5;
 
-  static const String CPU_COLOR = 'rgba(100, 181, 246, 1)'; // #64B5F6
-  static const String GPU_COLOR = 'rgba(77, 182, 172, 1)'; // #4DB6AC
+  static String CPU_COLOR =
+      'rgba(${mainCpuColor.red}, ${mainCpuColor.green}, ${mainCpuColor.blue}, 1)';
```

Comment:
use the existing colorToCss helper method instead.
Also in general prefer to call colorToCss right where colors are needed in code so as much of the logic stays in terms of the typed color objects.

Reply:
Done.